solidity/contracts/libs/StaticAddressSetFactory.sol

```diff
@@ -86,13 +86,45 @@ abstract contract StaticMOfNAddressSetFactory {
      * @return _bytecode The metaproxy bytecode used in Create2
      */
     function _saltAndBytecode(address[] calldata _values, uint8 _threshold)
-        private
+        internal
         view
         returns (bytes32, bytes memory)
     {
         bytes memory _metadata = abi.encode(_values, _threshold);
-        bytes memory _bytecode = MetaProxy.bytecode(_implementation, _metadata);
+        bytes memory _bytecode = MetaProxy.bytecode(implementation, _metadata);
         bytes32 _salt = keccak256(_metadata);
         return (_salt, _bytecode);
     }
 }
+
+abstract contract StaticAddressSetFactory is StaticMOfNAddressSetFactory {
```

Comment:
I kinda liked `StaticNAddressSetFactory` as it is really clear how it differentiates from `StaticMOfNAddressSetFactory`, but don't feel strongly at all